modules/nf-core/manta/convertinversion/environment.yml

```diff
@@ -1,9 +1,11 @@
 name: manta_convertinversion
+
```

Comment:
why the empty lines?

Reply:
That's `prettier`'s fault :grin:

Reply:
it hasn't happened in your other modules `environment.yml` file.
check modules/nf-core/delly/call/environment.yml
I'd rather we are consistent and not have useless empty lines

Reply:
Huh that's weird... I'll remove them here then